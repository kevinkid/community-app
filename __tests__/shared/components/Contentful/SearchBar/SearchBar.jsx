import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import { SearchBarInner } from 'components/Contentful/SearchBar/SearchBar';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  const MOCK_PROPS = {
    selected: '',
    options: [],
    theme: {},
  };
  renderer.render(<SearchBarInner {...MOCK_PROPS} />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
