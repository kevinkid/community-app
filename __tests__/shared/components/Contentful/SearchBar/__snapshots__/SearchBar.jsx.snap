// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div>
  <div>
    <IconSearch
      height="16"
      viewBox="0 0 16 16"
      width="16"
      xmlns="http://www.w3.org/2000/svg"
    />
    <input
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="Search.."
      type="text"
    />
    <div>
      <button
        onClick={[Function]}
        type="button"
      >
        <div>
          <IconFilterAll
            height="25"
            viewBox="0 0 25 25"
            width="25"
            xmlns="http://www.w3.org/2000/svg"
          />
          <span>
            All
          </span>
        </div>
        <IconDropdown
          height="12"
          viewBox="0 0 8 12"
          width="8"
          xmlns="http://www.w3.org/2000/svg"
        />
        <IconDropdownSmall
          className="undefined undefined"
          height="8"
          viewBox="0 0 12 8"
          width="12"
          xmlns="http://www.w3.org/2000/svg"
        />
      </button>
    </div>
  </div>
</div>
`;
